als-lsp/js/src/test/scala/org/mulesoft/lsp/conversions/ClientConversionTest.scala

```diff
@@ -148,26 +157,6 @@ class ClientConversionTest extends FlatSpec with Matchers {
 
   behavior of "Configuration transformations"
 
-  it should "transform ClientCapabilities" in {
```

Comment:
Why are we removing these tests?

Reply:
Our Custom Messages are not accessible from this module